src/fixate/ui_gui_qt/ui_gui_qt.py

```diff
@@ -266,9 +266,27 @@ def on_image_update(self, path):
             # message dialog so the user knows the image didn't load
             self.file_not_found(path)
         else:
-            image = QtGui.QPixmap()
-            image.loadFromData(image_data)
-            self.image_scene.addPixmap(image)
+            _, ext = os.path.splitext(path)
+            # Default to normal image if not a gif
+            if ext != ".gif":
+                image = QtGui.QPixmap()
```

Comment:
`ext.lower() != ".gif"` is probably better.

Also wondering if adding a new `user_ ` function would be a good idea? `user_gif` or `user_animated_gif`? Then there is no need to guess at what to do.

Reply:
Added the case-insensitive matching.
Can easily make it a separate user function - do we want it a part of the "overlaying" graphics scene? Don't know if that is ever used?

Reply:
I like the idea of an animated gif function, since regular gifs should still be able to work as normal images.
I was also thinking that if we don't use the overlay behaviour anywhere, then we should probably just ditch it since it requires just as much effort to size all the images properly to overlay them as it does just to overlay them manually in something like paint.

Reply:
Created separate user_gif() function.
Have left the image_scene for now and caught some bad cases when switching between gifs and images